packages/cli/openapi-ir-sdk/fern/definition/finalIr.yml

```diff
@@ -454,6 +455,7 @@ types:
       allOf: list<ReferencedSchema>
       properties: list<ObjectProperty>
       allOfPropertyConflicts: list<AllOfPropertyConflict>
+      extraProperties: boolean
```

Comment:
nit: i try to keep "openapi concepts" in the OpenAPI parser so calling these `additionalProperties` would make more sense and we map to `extra-properties` in openapi-ir-to-fern

Reply:
Totally makes sense, will update!